joeynmt/training.py

```diff
@@ -410,10 +417,11 @@ def train_and_validate(self, train_data: Dataset, valid_data: Dataset) \
                         self.scheduler.step(ckpt_score)
 
                     # append to validation report
-                    self._add_report(
-                        valid_score=valid_score, valid_loss=valid_loss,
-                        valid_ppl=valid_ppl, eval_metric=self.eval_metric,
-                        new_best=new_best)
+                    for idx, valid_score in enumerate(valid_scores):
+                        self._add_report(
```

Comment:
I'd rather pass multiple validation scores to _add_report to add more columns rather than rows to the validation score file.